geometries/intersection.json

```diff
@@ -0,0 +1,7 @@
+{
+	"id": 1108962113,
+	"name": "intersection",
```

Comment:
We have an `intersection` placetype json [here](https://github.com/whosonfirst/whosonfirst-placetypes/blob/master/placetypes/intersection.json).